arrow-libs/core/arrow-meta/arrow-meta-test-models/src/main/java/arrow/ap/objects/iso/IsoSealed.kt

```diff
@@ -6,7 +6,11 @@ import arrow.optics.OpticsTarget
 import arrow.optics.optics
 
 @optics([OpticsTarget.ISO])
-sealed class IsoSealed(val field: String, val nullable: String?, val option: Option<String>) {
-  data class IsoSealed2(val a: String?) : IsoSealed("", null, None)
-  companion object
+public sealed class IsoSealed(
+  public val field: String,
+  public val nullable: String?,
```

Comment:
These cannot be changed, and should be excluded from `explicitApi()`.
It's meant for testing kapt, and the tests rely on `public` not being there...

Changing this means we also have to update the tests, but kapt is actually deprecated but blocking by Arrow Meta not being released yet.

Reply:
This is the reason I enabled it: https://kotlinlang.slack.com/archives/C0B8MA7FA/p1632946470226500

I have to remove it from the published shared config until that is fixed.

Reply:
Fixed with your tip, thank you :)